config/local_config.properties

```diff
@@ -6,6 +6,7 @@ local_execution=False
 # General
 base_url=http://www.google.com
 driver=Firefox
+remote_hub=http://localhost:4444/wd/hub
```

Comment:
If you want to use remote webdriver instead of local - we need to provide also settings for OS name, because when user register node on hub - he should provide browsername and platform and not only browser name (java -jar selenium-server-standalone-2.44.0.jar -role webdriver -hub http://localhost:4444/grid/register -browser browserName=chrome,platform=WINDOWS - this exmaple of register node on hub)